src/snowcli/cli/snowpark/package/manager.py

```diff
@@ -59,15 +60,14 @@ def upload(file: Path, stage: str, overwrite: bool):
     return message
 
 
-def create(name: str):
-    file_name = name + ".zip"
-    if os.path.exists(".packages"):
-        utils.recursive_zip_packages_dir(pack_dir=".packages", dest_zip=file_name)
+def create(zip_name: str):
+    file_name = zip_name + ".zip"
```

Comment:
I'm wondering if it would make sens to do `file_name = zip_name if zip_name.endswith(".zip") else f"{zip_name}.zip"`. WDYT? It's quite common to pass full files names.

Reply:
Good point